Release 2.0.0 (#6)
voxpupuli/puppet-rkhunter: README.md

## Used Modules

> No external modules are used in this module. We rely on some times from stdlib.

Reviewer: Times? Should be types?

Reviewer: ```suggestion
Other than some data types from [puppetlabs/stdlib](https://forge.puppet.com/puppetlabs/stdlib), no other external modules are used.
```
